### ISSUE TITLE
Using TEST-SEAM ignores sy-subrc check

### ISSUE THREAD

**Issue by jan-jezek** [abaplint/abaplint#3108]
```abap
READ TABLE local_table INTO DATA(table_data).

IF sy-subrc <> 0.
  WRITE 'ERROR'.
ENDIF.
```
This case works fine, but when I add TEST-SEAM around the read table statement, check check_subrc rule is firing.
```abap
TEST-SEAM tst_seam.
  READ TABLE local_table INTO DATA(table_data).
END-TEST-SEAM.

IF sy-subrc <> 0.
  WRITE 'ERROR'.
ENDIF.
```
![image](https://github.com/abaplint/abaplint/assets/17530797/e4a9ec18-e4d9-4775-84d6-91a53ebfc52d)

I know that using TEST-SEAM is not recommended, but IMHO abaplint should not fire false positives.

**Comment by larshp** [comment]
@jan-jezek try again, 2.102.43 published
